### PR TITLE
Jq transformer

### PR DESCRIPTION
**Pull request by tibbycat**

### CODE REVIEW

**Comment by tlblessing** [comment]
fyi @eskibars 

Direct review link: https://deploy-preview-2411--kongdocs.netlify.app/hub/tibbycat/jq-transformer/

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:83]:
```suggestion
  service_id: true 
```

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:89]:
```suggestion
  route_id: true
```

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:96]: IS this plugin compatible with db-less mode? k8? Please see tabs for OSS and Enterprise here https://docs.konghq.com/hub/plugins/compatibility/

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:119]: Can you provide examples for request & response strings?

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:13]:
```suggestion

  <div class="alert alert-warning">
    <strong>Note: </strong> The Content-Type header will probably affect your results (`application/.*json` is probably the safest).
  </div>
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edits and requests for more information. We could add the install instructions and examples from the readme at the end of the file.

Marking as Do not merge for now until all information received and the plugin is approved for merging by @eskibars.

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tibbycat) [reply app/_hub/tibbycat/jq-transformer/index.md:99]: I am unsure of this as I don’t use dB-less mode. However, I don’t explicitly interact with the dB. And commonly use decK to apply the plugin.

Comment (tibbycat) [reply app/_hub/tibbycat/jq-transformer/index.md:99]: Would it be acceptable to say ‘partially’ as this ‘could be’ but is ‘untested’?

Comment (tlblessing) [reply app/_hub/tibbycat/jq-transformer/index.md:99]: Probably not but I'll defer to @eskibars

File: app/_hub/tibbycat/jq-transformer/index.md
Comment (tlblessing) [inline app/_hub/tibbycat/jq-transformer/index.md:134]:
```suggestion
    If you want to remove sensitive data such as {"name":"tibbycat","sex":"male"}, you could transform this using the expression {"name":.name}
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Another review comment.

**Comment by tlblessing** [comment]
Note to self: Clean out template instructions prior to merging.

**Comment by GayleNeumann** [comment]
@tibbycat
Thank you for the plugin submission for our Kong Hub. Unfortunately, we are not accepting new plugin entries at this time as we are in the process of revising our Plugin Hub and plugin submission criteria. We will have updates to our contributing guidelines soon and ask that you check back in a month or so. Again, thank you for your submission as we appreciate your efforts and contributions.  
Best, 
Gayle Neumann
Documentation Manager

**Pull request closed without merging**
